src/components/datepicker/datePicker.js

```diff
@@ -59,8 +65,10 @@
           '</md-button>' +
           '<div class="md-datepicker-input-container" ' +
               'ng-class="{\'md-datepicker-focused\': ctrl.isFocused}">' +
-            '<input class="md-datepicker-input" aria-haspopup="true" ' +
-                'ng-focus="ctrl.setFocused(true)" ng-blur="ctrl.setFocused(false)">' +
+            '<input class="md-datepicker-input begin" aria-haspopup="true" ' +
```

Comment:
All CSS classes must be prefixed, and ideally be self-explanatory. It should be `md-datepicker-start-date`.